packages/react-charting/UnitTests/__snapshots__/VerticalBarChartUT.test.tsx.snap

```diff
@@ -77,21 +77,21 @@ exports[`vertical bar chart with numeric x-axis data Should render the vertical
           <path
             class="domain"
             d="M56.5,6V0.5H614.5V6"
-            stroke="#000"
+            stroke="currentColor"
```

Comment:
Is this correct?

Reply:
Yes. This is part of a bug fix in d3 axis code. See https://github.com/d3/d3-axis/issues/49